app/src/main/java/com/example/pygmyhippo/MainActivity.java

```diff
@@ -23,6 +23,7 @@
 import com.example.pygmyhippo.databinding.OrganiserMainActivityNagivationBinding;
```

Comment:
has this been tested? i know you probably can't make junit test for this but is there a way to test this (ex hardcoding an event, manually doing something to send a notification, etc)?

Reply:
also, in your description, what does 'solutions to it' mean?